pkg/controller/plan/kubevirt.go

```diff
@@ -857,6 +859,48 @@ func (r *KubeVirt) GetGuestConversionPod(vm *plan.VMStatus) (pod *core.Pod, err
 	return
 }
 
+func (r *KubeVirt) GetVirtV2VDiskConfig(vm *plan.VMStatus, pod *core.Pod, step *plan.Step) (err error) {
```

Comment:
it's not really disk config, in oVirt we called it ["converted VM"](https://github.com/oVirt/ovirt-engine/blob/master/backend/manager/modules/vdsbroker/src/main/java/org/ovirt/engine/core/vdsbroker/vdsbroker/GetConvertedOvfVDSCommand.java)